Carbonate/Services/JsonSerializer.cs

```diff
@@ -16,7 +16,7 @@ internal sealed class JsonSerializer : ISerializer
     /// <inheritdoc/>
     public string Serialize<T>(T? value)
     {
-        // TODO: Set these options to only be indented if in debug mode
+        // TODO: Set these options to only be indented if in debug mode  ******Do you still need this TODO?********
```

Comment:
I do not need the TODO anymore.  I am going to push some changes on your branch to quickly fix this.